pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -480,6 +549,14 @@ def default_model_config(self) -> Dict:
             },
             "gamma_control": {"dist": "Normal", "kwargs": {"mu": 0, "sigma": 2}},
             "gamma_fourier": {"dist": "Laplace", "kwargs": {"mu": 0, "b": 1}},
+            "tvp_kwargs": {
+                "m": 200,
+                "L": None,
+                "eta_lam": 1,
+                "ls_mu": None,
+                "ls_sigma": 10,
+                "cov_func": None,
```

Comment:
What do we expect to happen with these `None` defaults?

Reply:
There's are checks starting on like 399, which set them if they are None. The reason that we wan't directly set them here is that the best defaults are estimated from the data, and we can't know here.